packages/cli/src/create/command.ts

```diff
@@ -10,14 +10,27 @@ import chalk from "chalk";
 import path from "path";
 import prompts from "prompts";
 
-let projectType: string = "";
 let projectPath: string = "";
+let projectType: string = "";
 let framework: string = "";
 let language: string = "";
 let baseContract: string = "";
+let chain: string = "";
 /* let createType: string = "app"; */
 
-export async function twCreate(pPath: string = "", options: any) {
+export async function twCreate(
+  pType: string,
+  pPath: string = "",
+  options: any,
+) {
+  if (pType === "app") {
+    projectType = "app";
+  }
+
+  if (pType === "contract") {
+    projectType = "contract";
+  }
+
```

Comment:
these need to also include `options.app` and `options.contract` respectively here

also they need to be `else` statements because otherwise I can do something like

```thirdweb create app --contract``` this should end up with projectType of `app` but since `--contract` gets evaluated afterwards and separately I will end up with projectType of `contract`

Reply:
also if and when we're in `app` vs `contract` mode the subsequent valid options change:
if I pass `projectType` of `contract`:
- `language` should now be ignored below
- `framework` should only be allowed to be the contract frameworks
... and the opposite for `app`